Dockerfile.build

```diff
@@ -0,0 +1,56 @@
+FROM node:6.11.3-alpine
+
+# phantomjs workaround
+RUN echo -e '#!/bin/sh\necho "2.1.1"' > /bin/phantomjs
+RUN chmod a+x /bin/phantomjs
+
+# bower workaround
+RUN echo '{ "allow_root": true }' > /root/.bowerrc
+
+# create release user so that we can chown output dir
+RUN adduser -u 663 -D release
+
+# install missing stuff
+RUN apk add --no-cache --virtual .gyp \
+        python \
+        make \
+        g++ \
+        git
+
+# create folder to cache dependencies
+WORKDIR /app
+COPY package.json .
+COPY bower.json .
+
+# install npm globals
+RUN npm install -g bower
+RUN npm install -g ember-cli
```

Comment:
How about using `npm run setup-for-local`? We should lock the versions and it's done there.

Reply:
we don't have it in `mobile-wiki` but thanks, worth considering :)